Header header
time src_time
string src_frame

Skeleton[] skeletons
